### PR TITLE
Update platform.sh

### PR DESCRIPTION
**Pull request by kotnik**
* [x] Description of Organization
* [x] Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

Description of Organization
====

Organization Website: https://platform.sh

Platform.sh is modern PaaS hosting solution so teams can focus on building experience and not on managing infrastructure.

Reason for PSL Update
====

We worked hard to remove our main domain `platform.sh` from the PSL and this pull-request would enable us to achieve that goal. Now we only need four entities in the list. 

DNS Verification via dig
=======

```
dig +short TXT _psl.bc.platform.sh
"https://github.com/publicsuffix/list/pull/1061"
```
and

```
dig +short TXT _psl.eu.platform.sh
"https://github.com/publicsuffix/list/pull/1061"
```
and

```
dig +short TXT _psl.us.platform.sh
"https://github.com/publicsuffix/list/pull/1061"
```

and

```
dig +short TXT _psl.ent.platform.sh
"https://github.com/publicsuffix/list/pull/1061"
```

make test
=========

Tests are green!

### CODE REVIEW

**Comment by dnsguru** [comment]
this is a revision to #417 that removes the previously requested suffix by dividing it into three segments.  makes sense.

**Comment by damz** [comment]
The PR for rockdaboot/libpsl has been merged.

**Comment by dnsguru** [comment]
I commented that the test fail that ocurred would be an expected outcome but LGTM.   I see @rockdaboot updated the test case (TYVM!) used, so it should probably be re-run.

@kotnik or @damz can you retest now that the tests were updated for this case?

**Comment by damz** [comment]
@dnsguru All green!

We didn't want to break your tests. After all, we are the ones requesting the changes.

File: public_suffix_list.dat [hunk 1]
```diff
@@ -12635,7 +12635,9 @@ on-web.fr
 
 // Platform.sh : https://platform.sh
 // Submitted by Nikola Kotur <nikola@platform.sh>
-*.platform.sh
+*.bc.platform.sh
```
Comment (pjcdawkins) [inline public_suffix_list.dat:12638]: Do you mean `bc.platform.sh`, etc?

File: public_suffix_list.dat [hunk 2]
```diff
@@ -12635,7 +12635,9 @@ on-web.fr
...
+*.eu.platform.sh
+*.us.platform.sh
```
Comment (pjcdawkins) [inline public_suffix_list.dat:12640]:
```suggestion
bc.platform.sh
ent.platform.sh
eu.platform.sh
us.platform.sh
```
Adding `ent.` and removing the wildcard: customer sites are directly below these 4 suffixes.

Comment (kotnik) [reply public_suffix_list.dat:12640]: You are right, @pjcdawkins, updating this PR now.

**Comment by kotnik** [comment]
We chased down, and added, the last of the `platform.sh` segments that should be in the PSL. This is now ready to go in. Thank you all.

**Comment by kotnik** [comment]
Please let me know if there is something else I can do to make sure this PR is merged? Thanks!

**Comment by dnsguru** [comment]
@kotnik Re-review needed - Leaving some time for any additional changes.  Typically the PR merge is final before submission, the atypical case of changes after approvals occur.

**Pull request merged**